src/Backend/Session.vala

```diff
@@ -41,9 +41,13 @@ public class Mail.Backend.Session : Camel.Session {
     construct {
         Camel.init (E.get_user_data_dir (), false);
         accounts = new Gee.LinkedList<Account> ();
-        set_network_monitor (E.NetworkMonitor.get_default ());
```

Comment:
It looks like @tintou added this in. I wonder if we can get a comment from him here?

Reply:
It is what Evolution itself is doing https://gitlab.gnome.org/GNOME/evolution/-/blob/master/src/libemail-engine/e-mail-session.c#L1250 unsure why we don't want it here?

Reply:
I'm not sure either so I added it back :)